packages/mapviewer/package.json

```diff
@@ -0,0 +1,123 @@
+        "update:translations": "node scripts/generate-i18n-files.js",
+        "check:external": "npx vite-node scripts/check-external-layers-providers.js",
+        "delete:reports": "rimraf tests/results/ || true",
+        "delete:reports:unit": "rimraf tests/results/unit/ || true"
+    },
+    "dependencies": {
+        "@fortawesome/fontawesome-svg-core": "^6.7.2",
+        "@fortawesome/free-brands-svg-icons": "^6.7.2",
+        "@fortawesome/free-regular-svg-icons": "^6.7.2",
+        "@fortawesome/free-solid-svg-icons": "^6.7.2",
+        "@fortawesome/vue-fontawesome": "^3.0.8",
+        "@geoblocks/cesium-compass": "^0.5.4",
+        "@geoblocks/mapfishprint": "^0.2.19",
+        "@geoblocks/ol-maplibre-layer": "^1.0.3",
+        "@ivanv/vue-collapse-transition": "^1.0.2",
+        "@mapbox/togeojson": "^0.16.2",
+        "@popperjs/core": "^2.11.8",
+        "@turf/turf": "^7.2.0",
+        "animate.css": "^4.1.1",
+        "axios": "^1.7.9",
+        "bootstrap": "^5.3.3",
+        "cesium": "1.119.0",
+        "chart.js": "^4.4.7",
+        "chartjs-plugin-zoom": "^2.2.0",
+        "dompurify": "^3.2.4",
+        "file-saver": "^2.0.5",
+        "form-data": "^4.0.1",
+        "geoadmin": "file:../geoadmin",
+        "geographiclib-geodesic": "^2.1.1",
+        "geotiff": "^2.1.3",
+        "hammerjs": "^2.0.8",
+        "jquery": "^3.7.1",
+        "jszip": "^3.10.1",
+        "lodash": "^4.17.21",
+        "maplibre-gl": "^5.1.0",
+        "ol": "^10.4.0",
+        "pako": "^2.1.0",
+        "print-js": "^1.6.0",
+        "proj4": "^2.15.0",
+        "reproject": "^1.2.7",
+        "sortablejs": "^1.15.6",
+        "tippy.js": "^6.3.7",
+        "vue": "^3.5.13",
+        "vue-chartjs": "^5.3.2",
+        "vue-i18n": "^11.1.0",
+        "vue-router": "^4.5.0",
+        "vue3-social-sharing": "^1.3.1",
+        "vuex": "^4.1.0"
+    },
+    "devDependencies": {
+        "@4tw/cypress-drag-drop": "^2.3.0",
+        "@cypress/vite-dev-server": "^6.0.2",
+        "@cypress/vue": "^6.0.2",
+        "@nuintun/qrcode": "^4.1.13",
+        "@prettier/plugin-xml": "^3.4.1",
+        "@rushstack/eslint-patch": "^1.10.5",
+        "@vitejs/plugin-basic-ssl": "^1.2.0",
+        "@vitejs/plugin-vue": "^5.2.1",
+        "@vue/eslint-config-prettier": "^10.2.0",
+        "@vue/eslint-config-typescript": "^14.3.0",
+        "@vue/tsconfig": "^0.7.0",
+        "axios-retry": "^4.5.0",
+        "chai": "^5.1.2",
+        "cypress": "^14.0.2",
+        "cypress-browser-permissions": "^1.1.0",
+        "cypress-real-events": "^1.14.0",
+        "cypress-recurse": "^1.35.3",
+        "cypress-vite": "^1.6.0",
+        "cypress-wait-until": "^3.0.2",
+        "dotenv": "^16.4.7",
+        "git-describe": "^4.1.1",
+        "gitconfig": "^2.0.8",
+        "googleapis": "^144.0.0",
+        "jsdom": "^26.0.0",
+        "mime-types": "^2.1.35",
+        "mocha-junit-reporter": "^2.2.1",
+        "rimraf": "^6.0.1",
```

Comment:
Maybe at some point we could do some cleanup with the dependencies? I quickly searched `rimraf` in the repo and didn't find any user of this, like no `import` or anything.

Reply:
it's used to delete JUnit test reports in an agnostic way (so that we may also have people developing on win11 for instance)